### PR TITLE
Adding known issue for backup and restore 2.5

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/25150

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -997,6 +997,22 @@ When you restore the settings for the `local-cluster` managed cluster resource a
 
 You must manually apply any configuration changes that are related to the `local-cluster` resource on the restored cluster. See link:../clusters/backup_and_restore.adoc#prepare-new-hub[Prepare the new hub cluster] for more details. 
 
+[#preparedforbackup-called-when-velero-schedules-are-created]
+=== prepareForBackup is only called when Velero schedules are first created
+//2.5.1:24012
+
+Any annotations defined in the `prepareForBackup` parameter is not functioning for resources created after the schedules creation. This affects Hive and AI secrets that are being annotated before the backup is initiated.
```
Comment (sahare) [inline release_notes/known_issues.adoc]: Should be: Any **labels** defined in the `prepareForBackup` **function** is not **added** to resources created after the schedules creation.
All other occurrences of annotation/annotated in the text should be updated to label/labeled
To clarify: prepareForBackup is a function which is called in the operator code before starting the backup process to add specific labels to some resources to include them in the backup, when these labels are missing these specific resources won't be backed up.

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -997,6 +997,22 @@ When you restore the settings for the `local-cluster` managed cluster resource a
...
+
+View the list of affected resources:
+
+* Secrets that are used by `clusterDeployments` and created by cluster claims
+* Cluster pool secrets
+* Secrets with the labels, `agent-install.openshift.io/watch` and `environment.metal3.io`
+
+Update the `BackupSchedule`, `veleroSchedule`, or `veleroTTL` values to start a new set of schedules. Then use the resulted backup for a restore, which is defined to annotate the latest resources for the backup.
+//What's AI? Assisted installer? If yes, that is currently listed as Infrastructure Operator for Red Hat OpenShift for our conref {assist-install}
```
Comment (sahare) [inline release_notes/known_issues.adoc]: Right, it's Assisted installer

**Review by sahare — CHANGES_REQUESTED** [review]
Thanks Mikela, the text that was mentioned in the issue was not clear, I think applying these changes will improve it.

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -997,6 +997,20 @@ When you restore the settings for the `local-cluster` managed cluster resource a
 
 You must manually apply any configuration changes that are related to the `local-cluster` resource on the restored cluster. See link:../clusters/backup_and_restore.adoc#prepare-new-hub[Prepare the new hub cluster] for more details. 
 
+[#preparedforbackup-called-when-velero-schedules-are-created]
+=== prepareForBackup is only called when Velero schedules are first created
+//2.5.1:24012
+
+Any labels defined in the `prepareForBackup` function is not added to resources created after the schedules creation. This affects Hive and {assist-install} secrets that are being annotated before the backup is initiated.
```
Comment (sahare) [inline release_notes/known_issues.adoc]: secrets that are being labeled before

File: release_notes/known_issues.adoc [hunk 4]
```diff
@@ -997,6 +997,20 @@ When you restore the settings for the `local-cluster` managed cluster resource a
...
+
+View the list of affected resources:
+
+* Secrets that are used by `clusterDeployments` and created by cluster claims
+* Cluster pool secrets
+* Secrets with the labels, `agent-install.openshift.io/watch` and `environment.metal3.io`
+
+Update the `BackupSchedule`, `veleroSchedule`, or `veleroTTL` values to start a new set of schedules. Then use the resulted backup for a restore, which is defined to annotate the latest resources for the backup.
```
Comment (sahare) [inline release_notes/known_issues.adoc]: which is defined to label the latest resources

**Review by sahare — CHANGES_REQUESTED** [review]
please update annotation occurrences  with label

File: release_notes/known_issues.adoc (hunk 3 above)
Comment (oafischer) [inline release_notes/known_issues.adoc]: ...are not added to...

**Review by oafischer — APPROVED** [review]
Looks good, thanks. Just a small typo. Approving.

**Comment by dockerymick** [comment]
Thanks for the reviews yall! Merging

**Pull request merged**